.github/workflows/docker.yml

```diff
@@ -20,9 +20,26 @@ jobs:
         uses: actions/checkout@v2
         with:
           java-version: 1.8
+          fetch-depth: 0
+      -
+        name: Setup GitVersion action
```

Comment:
can you please tell how will be mention Major and Minor? So we need to mention in PR name?

Reply:
i read your comment and it seems like we need to mention in commit message.

Reply:
can you please update the Readme with what we need to mention for major release and minor release? Also with example explain what is the impact of major release, minor release and Patch in the github tag and docker version ?

Reply:
ok will get this done

Reply:
@merry-degaga I merged this PR and I see that it has done a github release for the project where as i was thinking that it will just update the github tag and we will do the release as per our will.
https://github.com/SasanLabs/VulnerableApp/releases


is there a way we can stop it to create a release? From release, I point it as a major change so we notify about it in multiple channels.

Reply:
I deleted the release for now.

Reply:
Okay, I will take a look.

> @merry-degaga please help with [#382 (comment)](https://github.com/SasanLabs/VulnerableApp/pull/382#discussion_r945217490)

On OWASP-Vulnerableapp's DockerHub, I do see the updated version tag (1.11.0) and latest tags both attached to the most recently built image. Was there any other functionality you needed?

I will look into preventing auto-release creation.

Reply:
I was able to prevent auto-release creation.